tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -573,4 +574,99 @@ spec:
 			helper.CmdShouldPass("odo", "service", "delete", "EtcdCluster/etcd", "-f")
 		})
 	})
+
+	Context("Operators are installed in the cluster", func() {
+
+		JustBeforeEach(func() {
+			// wait till odo can see that all operators installed by setup script in the namespace
+			odoArgs := []string{"catalog", "list", "services"}
+			operators := []string{"etcdoperator", "service-binding-operator"}
+			for _, operator := range operators {
+				helper.WaitForCmdOut("odo", odoArgs, 5, true, func(output string) bool {
+					return strings.Contains(output, operator)
+				})
+			}
+		})
+
+		JustAfterEach(func() {
+			helper.DeleteProject(commonVar.Project)
+		})
+
+		Context("a specific operator is installed", func() {
+			var etcdOperator string
+			var etcdCluster string
+
+			JustBeforeEach(func() {
+				operators := helper.CmdShouldPass("odo", "catalog", "list", "services")
+				etcdOperator = regexp.MustCompile(`etcdoperator\.*[a-z][0-9]\.[0-9]\.[0-9]-clusterwide`).FindString(operators)
+				etcdCluster = fmt.Sprintf("%s/EtcdCluster", etcdOperator)
+			})
+
+			When("a nodejs component is created", func() {
+
+				JustBeforeEach(func() {
+					helper.CmdShouldPass("odo", "create", "nodejs")
```

Comment:
Please use `--context` for this and the other commands below.

Reply:
Why is `--context` needed?

In this scenario, all the commands are executed in the same directory and so I think the flag is not necessary.

Reply:
Just to make sure that the commands are triggered in the desired directory during parallel runs.

Reply:
There were no `--context` flags in `cmd_service_test.go` before thir PR, nor before the refactoring of this file. I would prefer not to add it.

Reply:
OK, let's leave it for now.